.github/actions/cargo-install-upload-artifacts/action.yml

```diff
@@ -29,17 +29,15 @@ runs:
       shell: bash
 
     - name: Build with all features
-      uses: actions-rs/cargo@v1
-      with:
-        command: install
-        args: >
-          --locked
-          --path .
-          --target ${{ inputs.target }}
-          --all-features
-          --root ${{ steps.metadata.outputs.out-dir }}
+      run: |
+        cargo install \
+          --locked \
+          --path . \
+          --target ${{ inputs.target }} \
+          --all-features \
+          --root ${{ steps.metadata.outputs.out-dir }} \
           --bins
-        use-cross: true
+      shell: bash
```

Comment:
I think this is the issue, actually. This is causing the following issue in our actions, as shown [here](https://github.com/cross-rs/cross/actions/runs/3168710823/jobs/5160122507#step:13:57):

It's installing `cross` to `C:UsersRUNNER~1AppDataLocalTemptmp.FKQJyKb3ZP\bin\cross.exe`, `outdir` is `/tmp/tmp.FKQJyKb3ZP`. In previous [runs](https://github.com/cross-rs/cross/actions/runs/3164781446/jobs/5153342326#step:13:57), `cross` wwas installed to `C:\Users\RUNNER~1\AppData\Local\Temp\tmp.2K6uluIXkp\bin\cross.exe` and had the outdir be `/tmp/tmp.2K6uluIXkp`.

In short, `shell: bash` is undoing the `\` characters for our paths.